test/spec.test.js

```diff
@@ -65,36 +65,28 @@ skywalker.on('end', function() {
     var testFile = readCleanCRLF(testFilePath);
     var source   = readCleanCRLF(sourcePath);
     const actual = dogescript(source, true);
-    
+
+
     var testName = getFolderName(dir);
-    
     // separate by directory if it wasnt a directory with a name-like-this
     if(!testName.includes('-'))
     {
       var relative = path.relative(specDir, dir);
       testName = relative.split(path.sep).join('-');
     }
-    
+
     test(testName, function(t) {
       t.plan(1);
-        
+
       var skip = fs.existsSync(path.join(dir, 'skip'));
       if(skip)
       {
         t.skip('skipped');
         return;
       }
-      
+
       if (fs.existsSync(path.join(dir, 'expect.js')))
       {
-        var testFilePath   = path.join(dir, 'expect.js');
```

Comment:
What is this change?  Why is it part of this branch?

Reply:
coolios, reset!